src/ucp/proto/proto_init.c

```diff
@@ -339,22 +339,30 @@ void ucp_proto_init_memreg_time(const ucp_proto_common_init_params_t *params,
         return;
     }
 
-    perf_node = ucp_proto_perf_node_new_data("mem reg", "");
+    if (context->rcache != NULL) {
+        perf_node = ucp_proto_perf_node_new_data("lookup", "");
 
-    /* Go over all memory domains */
-    ucs_for_each_bit(md_index, reg_md_map) {
-        md_attr = &context->tl_mds[md_index].attr;
-        md_name = context->tl_mds[md_index].rsc.md_name;
-        ucs_linear_func_add_inplace(memreg_time, md_attr->reg_cost);
-        ucs_trace("md %s" UCP_PROTO_PERF_FUNC_FMT(reg_cost), md_name,
-                  UCP_PROTO_PERF_FUNC_ARG(&md_attr->reg_cost));
+        *memreg_time = ucs_rcache_lookup_cost();
 
-        ucp_proto_perf_node_add_data(perf_node, md_name, md_attr->reg_cost);
-    }
+        ucp_proto_perf_node_add_data(perf_node, "lookup", *memreg_time);
```

Comment:
Do we have a place where we define labels for protocol selection tree?
If not so, I think it may be a good thing to start defining all the labels in one place. (I mean "lookup", "mem reg" ...).

And could you please attach a screenshot of the protocol selection tree with the new node to better understand how the new names are used.

Reply:
Here is the screenshot:
![image](https://user-images.githubusercontent.com/22097249/229750399-9139b26f-ebd5-4bdf-94d2-25d8447d6223.png)


Will take a look for a place for labels little-bit later

Reply:
The only place that I can suggest is the field inside `ucp_context_t` structure, but I don't think that this is a good idea.

Reply:
Sorry, my comment was not very clear.
I meant that it is better to have a place where we define labels for the tree:
```C
/* proto_debug.h, or create new proto_debug_def.h */
#define UCP_PROTO_PERF_NODE_RCACHE_LOOKUP      "rcache lookup"
#define UCP_PROTO_PERF_NODE_DATA_RCACHE_LOOKUP "lookup"

/* proto_init.c */
perf_node    = ucp_proto_perf_node_new_data(UCP_PROTO_PERF_NODE_RCACHE_LOOKUP, "");
*memreg_time = ucs_rcache_lookup_cost();
ucp_proto_perf_node_add_data(perf_node, UCP_PROTO_PERF_NODE_DATA_RCACHE_LOOKUP, *memreg_time);
```

Reply:
It was decided to implement it for all the cases in the separate PR.